private-media-framework/draft-ietf-perc-private-media-framework.md

```diff
@@ -487,10 +485,34 @@ Key Distributor and endpoints will also be used to convey HBH key
 information from the Key Distributor to the Media Distributor, so no
 additional protocol or interface is required.
 
+In establishing the DTLS association between endpoints and the
+Key Distributor, the endpoint acts as the DTLS client and the
+Key Distributor acts as the DTLS server.  The Key Encryption Key (KEK)
+(i.e., EKT Key) is conveyed by the Key Distributor over the DTLS
+association to endpoints via procedures defined in PERC EKT
+[I-D.ietf-perc-srtp-ekt-diet] via the EKTKey message.  Endpoints do not
+send EKTKey messages over the DTLS association and the Key Distributor
+**MUST** silently discard any EKTKey messages received from an endpoint.
```

Comment:
If we were going to say this, we should say it in the EKT doc.  But we shouldn't say it, because it's wrong :)  DTLS clients are not allowed to send EKTKey, so the server should return an alert if one does.

Reply:
There is no Key Distributor in the EKT document.  It has to be put in the framework.

Reply:
Doesn't matter.  You've already said that the KD is the server and the endpoint is the client, and EKT forbids the client from sending EKTKey.  So you're just being duplicative here.

Reply:
I do not follow.  We cannot put this text into the EKT Diet document.  And, this is the only instance in the text that uses the phrase "DTLS server".

Reply:
Oh, I understand.  You're saying the last sentence isn't needed, because it implies direction on who can send what.